MIP104/MIP104.md

```diff
@@ -883,7 +883,7 @@ Current WBTC-A parameters are:
 
 Current WBTC-B parameters are:
 
-- Stability Fee: 15.25%
+- Stability Fee: 10.30%
```

Comment:
```suggestion
- Stability Fee: 10.25%
```

Reply:
![image](https://github.com/makerdao/mips/assets/139177554/7bcb108d-3355-4d7f-b103-a14f1cd05b22)

![image](https://github.com/makerdao/mips/assets/139177554/a5396f31-7e09-46dd-820c-dfb45ec4f81e)

Good catch, Makerburn was rounding 10.25 - see image. We will report this to BA Labs as the rounding seems to be inconsistent - as 10.75 is not rounded to 10.8 on the same page.